### DISCUSSION TITLE
I have an email in use but it dosen't show with the primary email

### DISCUSSION THREAD

**Discussion by clau-o** [community/community#58262] [Other Feature Feedback, Questions, & Ideas] [answered]
HELP PLEASEE 

Hi everyone,

I have an issue with one of my emails in github.
I have 2 email in github, one that is gmail and the other one that is from my uni.
![image](https://github.com/community/community/assets/124678589/18b7b6a0-ea1b-4f64-ba0c-14fa9a3eacbf)

This first image shows that my gmail account is correctly in github, however, for me to have there my uni account too, when I insert the mail in the textbox and click 'Add', it shows the following message:

![image](https://github.com/community/community/assets/124678589/bd2a0b32-56f4-4ed0-bac2-fe76200fa02a)

However, I cant find it anywhere, but it still says that my uni email is already in use. 
I need help because I need to be able to use my uni email connected to github.
Thank you all in advance 👍 :)

**Comment by ALSIHMAN** [comment]
04496511

**Reply by ALSIHMAN** [reply]
Ns.444@hotmail.com.

**Reply by ALSIHMAN** [reply]
> 04496511

Nd.4444@outooll.s

**Comment by the-officialjosh** [comment, marked as answer]
:

I understand that you're experiencing difficulties adding your university email to your GitHub account. To resolve this issue, I recommend following these steps:

1. Double-check your email address: Make sure you have entered your university email address correctly in the textbox. Verify that there are no typos or extra spaces.

2. Check for existing accounts: It's possible that you may have already created a GitHub account using your university email. Try using the ["Forgot password?"](https://github.com/password_reset) option on the GitHub login page with your university email address to check if an account exists. If you find an existing account, you can reset the password and gain access to it.

3. Contact GitHub support: If you're unable to find an existing account associated with your university email or if you're still encountering difficulties, I recommend reaching out to [GitHub support](https://support.github.com/contact). They will be able to assist you in resolving the issue and adding your university email to your GitHub account.

To contact GitHub support, visit the [GitHub Support](https://support.github.com/) page and follow the instructions to submit a request or reach out to their support team through their official communication channels.

They will have access to your account details and can provide you with specific guidance and assistance to resolve the issue.

I hope this helps, and good luck with resolving the email addition problem!

**Reply by clau-o** [reply]
Thank you very much, I'll try that now

**Reply by marmarcosflores** [reply]
How do you put your school account in "not use" to use it as secondary?

**Comment by Ropsten80** [comment]
HI

**Reply by ghostinhershell** [reply]
👋 Welcome to the GitHub Community on Discussions, @Ropsten80! Happy to have you here 🙂

To get started, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776).
